src/extension.ts

```diff
@@ -463,8 +470,13 @@ function enableScaladocIndentation() {
   });
 }
 
-function dottyIdeArtifact(): string {
-  return path.join(workspace.rootPath, ".dotty-ide-artifact");
+function dottyIdeArtifact(): string | undefined {
+  if (workspace.workspaceFolders) {
```

Comment:
I've also started using `workspaceFolders` instead of the deprecated `rootPath`. As per the documentation `rootPath` == `worspaceFolders[0]`